api/api/common/config/auth.py

```diff
@@ -18,6 +18,8 @@
 COGNITO_RESOURCE_SERVER_ID = f"https://{DOMAIN_NAME}"
 COGNITO_USER_POOL_ID = os.environ["COGNITO_USER_POOL_ID"]
 
+CUSTOM_USERNAME_REGEX = os.environ.get("CUSTOM_USERNAME_REGEX")
```

Comment:
Don't think this wants to sit in the auth file, instead probably wants to move to the constants file